datadog-lambda.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |spec|
   spec.email                 = ['dev@datadoghq.com']
 
   spec.summary     = 'Instruments your Ruby AWS Lambda functions with Datadog'
-  spec.description = <<-MSG.gsub(/^[\s]+/, '')
+  spec.description = <<-MSG.gsub(/^\s+/, '')
```

Comment:
automatic change

 Redundant single-element character class, `[\s]` can be replaced with `\s`.Style/RedundantRegexpCharacterClass